heron/proto/ckptmgr.proto

```diff
@@ -171,7 +175,23 @@ message CleanStatefulCheckpointResponse {
   repeated string cleaned_checkpoint_ids = 2;
 }
 
-// stmgr -> ckptmgr messages
+/*
+ * stmgr -> ckptmgr messages
+ */
+
+// This message encapsulates the info associated with
+// state of an instance/partition
+message InstanceStateCheckpointPartition {
```

Comment:
Is this just instance or partition or instance and partition? Its unclear, especially since it does not have anything to do with partition right now

Reply:
My suggesstion would be to leave the name of InstanceStateCheckpoint for now and rename later?

Reply:
kk. let me do it.